PHPCI/Builder.php

```diff
@@ -287,7 +283,7 @@ public function interpolate($input)
      */
     protected function setupBuild()
     {
-        $this->buildPath = PHPCI_DIR . 'PHPCI/build/' . $this->build->getId() . '/';
+        $this->buildPath = $this->build->getBuildPath() . '/';
```

Comment:
All platforms support / as a directory separator, and it is a lot easier to read if you use do it this way.

The only times you need to use DIRECTORY_SEPARATOR are (a) for display purposes, so you output in the expected format for the local OS; or (b) if you're doing a comparison between paths, where one of the paths is in local format (though in this case you're safest using realpath() on both of your inputs first, to resolve any symlinks, "/../" elements, etc. and that will tidy any separators for you anyway).

Therefore it is much better to use a literal / instead of the long winded DIRECTORY_SEPARATOR unless the path is going to be shown to the user, in which case tidying at point-of-output might be the most sensible approach anyway.